Platform/ElkhartlakeBoardPkg/AcpiTables/Dsdt/Pep.asl

```diff
@@ -871,7 +872,8 @@ Scope(\_SB)
           If(LEqual(S0ID, 1)) { //S0ID: >=1: CS 0: non-CS
             // call method specific to CS platforms when the system is in a
             // standby state with very limited SW activities
-            \_SB.PC00.SPIF.SPIS() - Clear SPI Synchronous SMI Status bit
+            \_SB.PC00.SPIF.SPIS() // Clear SPI Synchronous SMI Status bit
+                  Store(0x0000000000000000, \_SB.PC00.LPCB.HPET.TCN1)

```

Comment:
Please fix the indentation.
Also, did you check if this TCN1 register will be reprogrammed after you exit S0ix by OS ?   
Otherwise, HPET based function might not work properly after S0ix.

Reply:
From TGL check using pythonsv, whatever the value of TIMER1_CNP before S0ix the same value is present after S0ix exit.

Reply:
Indentation fixed.
Yes TCN1 register is same before and after s0ix entry.
Verified with pythonsv command pch.itss.hpet.mem.tmr1_cnf_cap.